src/Stripe.net/Entities/Disputes/Dispute.cs

```diff
@@ -5,11 +5,8 @@ namespace Stripe
     using Newtonsoft.Json;
     using Stripe.Infrastructure;
 
-    public class Dispute : StripeEntityWithId, ISupportMetadata
+    public class Dispute : StripeEntityWithIdAndObject
```

Comment:
I don't think Discount has an id, at least it's not in the docs.

Reply:
👍  Fixed.